### PR TITLE
[Cache] Mention that user/password are not supported for Redis DSN

### LINKED ISSUE #19955: Add support for ACL auth in RedisAdapter, for 5.x branch
### Symfony version(s) affected

5.x

### Description

As I'm using `symfony/cache:v5` (in a project that requires PHP 7.2, unfortunately), I'm facing [this issue](https://github.com/symfony/symfony/issues/45305).

The [Symfony 5 documentation](https://symfony.com/doc/5.x/components/cache/adapters/redis_adapter.html#configure-the-connection) states that it is possible to pass an user/password pair: 

```
redis[s]:[[user]:pass@]?[ip|host|socket[:port]][&params]
```

I understand, with [this comment](https://github.com/symfony/symfony/pull/45313#pullrequestreview-873247940), that it has been considered as a new feature, and not a fix.

Should it be possible  to add this fix also in the 5.x branch ?

If not, I guess the documentation should be updated for clarification.

### How to reproduce

Same as https://github.com/symfony/symfony/issues/45305, but with `symfony/cache:v5`

### Possible Solution

Use this patch : https://github.com/symfony/symfony/pull/45313/commits/722830d3c658c0652e95b4720da83b0f35091796#diff-38029e8e3e92a47fb8091876d3e8bc7fc61c2fc793cabf842ed933bf6985134c

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #19955.

### CODE REVIEW

File: components/cache/adapters/redis_adapter.rst [hunk 1]
```diff
@@ -55,23 +55,25 @@ helper method allows creating and configuring the Redis client class instance us
...
+be replaced by ``rediss`` (the second ``s`` means "secure").
 
 .. note::
 
     A `Data Source Name (DSN)`_ for this adapter must use either one of the following formats.
 
     .. code-block:: text
 
-        redis[s]://[pass@][ip|host|socket[:port]][/db-index]
+        redis[s]://[ip|host|socket[:port]][/db-index]
```
Comment (xabbuh) [inline components/cache/adapters/redis_adapter.rst]: if I read the changes from symfony/symfony#45313 correctly, passing a password was possible before. Then reverting #17234 seems to be the solution to me.

Comment (javiereguiluz) [reply components/cache/adapters/redis_adapter.rst]: OK. But this revert must be applied only to 5.4, right? 6.4 and up should be kept unchanged.

Comment (xabbuh) [reply components/cache/adapters/redis_adapter.rst]: yes, that's how I see it as the support for this form was introduced in Symfony 6.1

Comment (javiereguiluz) [reply components/cache/adapters/redis_adapter.rst]: OK, I reverted everything that was added in #17234 ... and I'll readd it in 6.4 branch and up when merging this.

**Review by xabbuh — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
OK, this was merged ... and the changes reverted in 6.4 and up here: https://github.com/symfony/symfony-docs/commit/b7fe8d48ec7fc9c6e0fb450cde2e6e329ee030ac

**Pull request merged**